generators/generator-base-private.js

```diff
@@ -1384,10 +1384,26 @@ module.exports = class extends Generator {
      * @param {string} clientRootFolder
      */
     getEntityParentPathAddition(clientRootFolder) {
-        if (clientRootFolder) {
-            return '../';
+        if (!clientRootFolder) {
+            return '';
+        }
+        const relative = path.relative(`/app/entities/${clientRootFolder}/`, '/app/entities/');
```

Comment:
using the path api from node may be a better idea, don't you think?
one simple use case I can think of is if `clientRootFolder` starts or ends with a /.

Reply:
I don't think it is supposed to start with / here. But added more tests.

I couldn't get a simpler way to do this.